helper/secret_sharing.go

```diff
@@ -94,3 +94,36 @@ func GetShamirSharedRandomElement(rng *rand.Rand, t, n int) (*bls12381.Fr, []*bl
 	}
 	return secretKeyElement, shares
 }
+
+// GetShamirSharedRandomElement generates a t-out-of-n shamir secret sharing of a random element, using a provided secret key
```

Comment:
```suggestion
// ShamirSharedSecretKey generates a t-out-of-n shamir secret sharing of the provided secret key```

If I understand this function correctly, but maybe I misunderstand it?